models/issue.go

```diff
@@ -1568,25 +1569,34 @@ func SearchIssueIDsByKeyword(kw string, repoIDs []int64, limit, start int) (int6
 	return total, ids, nil
 }
 
-func updateIssue(e Engine, issue *Issue) error {
-	_, err := e.ID(issue.ID).AllCols().Update(issue)
-	if err != nil {
-		return err
-	}
-	return nil
+func updateIssueByCols(e Engine, issue *Issue, columns ...string) (err error) {
+	_, err = e.ID(issue.ID).Cols(columns...).Update(issue)
+	return
 }
 
-// UpdateIssue updates all fields of given issue.
-func UpdateIssue(issue *Issue) error {
+// UpdateIssueByAPI updates all allowed fields of given issue.
+func UpdateIssueByAPI(issue *Issue) error {
+	// allowed fields to update
+	columns := []string{"name", "is_closed", "content", "milestone_id",
+		"priority", "num_comments", "ref", "deadline_unix", "created_unix",
```

Comment:
`"num_comments", "ref","created_unix"` should not be changed by edit issue.

Reply:
I just add all colums wich could be updated (before PR) beside some colums wich realy should not be changed (repom & index)

... I'll minimize it and remove Poster completly